app/server/appsmith-server/src/test/java/com/appsmith/server/services/UserServiceTest.java

```diff
@@ -415,12 +415,36 @@ public void createUserWithInvalidEmailAddress() {
     @WithUserDetails(value = "api_user")
     public void updateNameOfUser() {
         UserUpdateDTO updateUser = new UserUpdateDTO();
-        updateUser.setName("New name of api_user");
+        updateUser.setName("New name of api user");
         StepVerifier.create(userService.updateCurrentUser(updateUser, null))
                 .assertNext(user -> {
                     assertNotNull(user);
                     assertThat(user.getEmail()).isEqualTo("api_user");
-                    assertThat(user.getName()).isEqualTo("New name of api_user");
+                    assertThat(user.getName()).isEqualTo("New name of api user");
```

Comment:
Let's prefer `assertEquals` and slowly move towards having more tests use that. Why? It produces better diffs and better logs when the test fails, so we can spot the problem easier.

```suggestion
                    assertEquals("New name of api user", user.getName());
```

Note that it takes the _expected value_ as first parameter, and the _actual value_ as the second parameter.